website/en/docs/types/objects.md

```diff
@@ -314,17 +318,29 @@ function method(obj: {}) {
 ```
 
 However, if you need to opt-out of the type checker, and don't want to go all
-the way to `any`, you can instead use `Object`. **`Object` is unsafe and
-should be avoided.**
+the way to `any`, you could use `{ [key: string]: any}`. ** NOTE [`any`](../any/) is unsafe and
+should be avoided.** For historical reasons, the `Object` keyword is still avaiable
+but is an alias to [`any`](../any/). In previous versions of Flow, `Object` was the same
+as `{ [key: string]: any}`.
```

Comment:
maybe say "was roughly the same" since there are subtle behaviour differences, e.g.:

``` 
function f(): Object {
  return {}
}
f() + 1
```
won't throw but:

```
function f(): { [key:string]: any } {
  return {}
}

f() + 1
```
will

Reply:
The former does seem to [produce an error in v0.88.0 and below][1], which I believe is what the paragraph is referring to about previous versions.

[1]: https://flow.org/try/#0PTAEAEDMBsHsHcBQjIFcB2BjALgS1uqJABQCUAXKAPIBGAVgKY6gDeiooATg9qp4SwC+iYSjKgA1KACMiIA